### DISCUSSION TITLE
Problem of chating

### DISCUSSION THREAD

**Discussion by Jones-Sw** [community/community#130403] [Actions] [answered]
### Select Topic Area

Question

### Body

Problem of chating

**Comment by Doenya1988** [comment, marked as answer]
Hallo zou iemand me mss kunnen helpen aan een paar paysafecard codes ik snap er niks van en krijg het zelf niet

**Comment by Doenya1988** [comment]
> ### Selecteer Onderwerpgebied
> Vraag
> 
> ### Lichaam
> Probleem met chatten

Paysafecard code prapaid

**Comment by grandmas-favorite** [comment]
Hi @Jones-Sw , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
